codes/classical/q-ary_digits/simplex.yml

```diff
@@ -0,0 +1,27 @@
+code_id: simplex
+physical: q-ary_digits
+logical: q-ary_digits
+
+name: 'Simplex code'
+
+description: |
+  An \([n,k,q^{k-1}]\) linear code over \(\mathbb{F}_q\), where \(n=\frac{q^k-1}{q-1}\). The columns of the generator matrix are in one-to-one correspondence with the elements of the projective space \(PG(k-1,q)\), with each column being a chosen representative of the corresponding element.
+
+features:
+
+  decoders:
+    - 'A quantum decoding algorithm \cite{manual:{A. Barg and S. Zhou, A quantum decoding algorithm of the simplex code, 2004.}}.'
```

Comment:
manual:{A. Barg and S. Zhou, “A quantum decoding algorithm for the simplex code”, in
Proceedings of the 36th Annual Allerton Conference on Communication, Control and
Computing, Monticello, IL, 23–25 September 1998 (UIUC 1998) 359–365}